build.gradle

```diff
@@ -61,6 +64,9 @@ dependencies {
 	// for config api
 	compile "tk.valoeghese:ZoesteriaConfig:1.3.0"
 
+	implementation name: "slick-util"
```

Comment:
why not use maven central

Reply:
More to the point, what is Slick-Util getting us that just using `TextureManager` directly wouldn't? Sure it could do fonts but this isn't using that and it's questionable to whether the API especially needs to be doing it anyway